app/lib/features/chat/widgets/message_metadata_builder.dart

```diff
@@ -5,10 +5,10 @@ import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
     show Convo, EventSendState;
 import 'package:atlas_icons/atlas_icons.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_riverpod/flutter_riverpod.dart';
-import 'package:quds_popup_menu/quds_popup_menu.dart';
 import 'package:flutter_chat_types/flutter_chat_types.dart' as types;
+import 'package:flutter_riverpod/flutter_riverpod.dart';
```

Comment:
you removed it everywhere else, but here you kept `flutter_riverpod` ... why?

Reply:
Provider may use `riverpod`, but UI should use `flutter_riverpod`.
I wanted to reduce the occurrence of `flutter_riverpod` usage as many as possible.